src/main/kotlin/io/arrowkt/repository/interpreter/AccountRepository.kt

```diff
@@ -0,0 +1,36 @@
+package io.arrowkt.repository.interpreter
+
+import arrow.core.Option
+import arrow.core.getOrElse
+import arrow.core.right
+import arrow.core.toOption
+import arrow.effects.IO
+import io.arrowkt.ErrorOr
+import io.arrowkt.model.Account
+import io.arrowkt.repository.AccountRepository
+import io.arrowkt.today
+import java.time.LocalDate
+
+open class AccountInMemoryRepository : AccountRepository {
```

Comment:
open just doesn't feel right here.... what would be a more kotlin solution?

Reply:
sealed class or interface

Reply:
Or do `object` directly.... the original interface is still there for other implementations